SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -1591,8 +1591,6 @@ void CNEMOEulerSolver::BC_Far_Field(CGeometry *geometry, CSolver **solver_contai
         /*--- Primitive variables, and gradient ---*/
         visc_numerics->SetConservative(nodes->GetSolution(iPoint),
                                        node_infty->GetSolution(0) );
-        visc_numerics->SetConsVarGradient(nodes->GetGradient(iPoint),
-                                          node_infty->GetGradient(0) );
```

Comment:
So these are unnecessary? Looks good to me otherwise, as long as this doesn't cause any issues.